Automattic/jetpack
# VideoPress: give access to secondary admins

Fixes https://github.com/Automattic/jetpack/issues/26940

#### Changes proposed in this Pull Request:

* VideoPress: give access to secondary admins:
  * check site connections to get purchases
  * change the site connection to be the only requirement to show the dashboard

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to the VideoPress dashboard
* Connect the user
* Confirm the user can use the VideoPress app
* In another session (incognito mode, another browser...)
* Do not connect the user, and confirm the site is connected

<img width="687" alt="Screen Shot 2022-10-20 at 12 25 50" src="https://user-images.githubusercontent.com/77539/196991516-3e2d22a8-9d08-4d56-82bf-5358ab12897c.png">

* Confirm you see the VideoPress dashboard, even when the user is not connected
* Confirm you can use the app (upload the video, etc.)
